docs/building.md

```diff
@@ -11,7 +11,7 @@ in particular its CameLIGO dialect.
 To generate the contract code, or the storage for one of the provided examples,
 you will need the [ligo executable](https://ligolang.org/docs/intro/installation) installed.
 
-The latest working version tested is [0.46.0](https://gitlab.com/ligolang/ligo/-/releases/0.46.0).
+The latest working version tested is [0.47.0](https://gitlab.com/ligolang/ligo/-/releases/0.47.0).
```

Comment:
This commit 9bb6a19d9e2009b263d71d3e5403e9d279c9601e only has this change, but its message mentions deprecations and warnings.

I also see that the LIGO versions used by CI hasn't been updated here.

So, I suppose that this is a rebase artifact, I'd suggest to drop this commit entirely.

Reply:
Dropped